packages/twenty-front/src/modules/dropdown/hooks/useCurrentContentId.ts

```diff
@@ -0,0 +1,19 @@
+import { useCallback, useState } from 'react';
+
+export const useCurrentContentId = <T>() => {
```

Comment:
Seems like we should create a component state v2 in the dropdown folder and that those two handlers should belong to a component, we generally avoid having handlers in hooks because handlers are tied to events and should be specific to each component. 

If we need to abstract some bit of logic, we can create a hook to abstract what's called inside the handler.

Reply:
I'll handle it in my PR